exporter/lokiexporter/next_exporter_test.go

```diff
@@ -133,3 +133,106 @@ func TestPushLogData(t *testing.T) {
 		})
 	}
 }
+
+func TestLogsToLokiRequestWithGroupingByTenant(t *testing.T) {
+	tests := []struct {
+		desc     string
+		logs     plog.Logs
+		expected map[string]struct {
+			line  string
+			label string
+		}
+	}{
+		{
+			desc: "create a push request per tenant",
+			logs: func() plog.Logs {
+				logs := plog.NewLogs()
+				rl := logs.ResourceLogs().AppendEmpty()
+
+				sl := rl.ScopeLogs().AppendEmpty()
+				logRecord := sl.LogRecords().AppendEmpty()
+				logRecord.Attributes().PutStr("loki.tenant", "tenant.id")
+				logRecord.Attributes().PutStr("tenant.id", "1")
+				logRecord.Attributes().PutInt("http.status", 200)
+
+				sl = rl.ScopeLogs().AppendEmpty()
+				logRecord = sl.LogRecords().AppendEmpty()
+				logRecord.Attributes().PutStr("loki.tenant", "tenant.id")
```

Comment:
Can you add a teste where the attribute isn't found? And one where the tenant is at the resources level.

Reply:
> And one where the tenant is at the resources level.

This one is already present in `pkg/loki`, but let's add it anyways.